### PR TITLE
Lorem-ipsum-bundle recipe test

### PR DESCRIPTION
**Pull request by BooleanType**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by OskarStark** [comment]
No files changed 🧐

**Comment by BooleanType** [comment]
> No files changed 🧐

Yeah, that's strange for me, 'cause I pushed them hour ago. Now re-pushed.

**Comment by BooleanType** [comment]
Also I don't get, why I still see "There are some validation errors", when click on "View deployment" link. I've fixed everything and all checks have passed.

image

File: boolean-type/lorem-ipsum-bundle/1.0/manifest.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "KnpU\\LoremIpsumBundle\\KnpULoremIpsumBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "KNPU_LOREM_IPSUM_FOO": "hello",
+        "KNPU_LOREM_IPSUM_BAR": "world"
```
Comment (Nyholm) [inline boolean-type/lorem-ipsum-bundle/1.0/manifest.json:10]: What are these environment variables for?

Comment (BooleanType) [reply boolean-type/lorem-ipsum-bundle/1.0/manifest.json:10]: It was only for test: I just wanted to see, if variables are set after bundle installation.
I will handle it soon if needed.

Comment (BooleanType) [reply boolean-type/lorem-ipsum-bundle/1.0/manifest.json:10]: Ok, I've decided not to change bundle code to use those variables. There is possible for user to use them instead. For example, in subscriber, which allows to change data returned by API:

<pre>
public static function getSubscribedEvents()
{
    return [
        KnpULoremIpsumEvents::FILTER_API => 'onFilterApi',
    ];
}

public function onFilterApi(FilterApiResponseEvent $event)
{
    $data = $event->getData();
    $data['message'] = 'Have a magical day!';
    $data['hello_world'] = $_ENV['KNPU_LOREM_IPSUM_FOO'] . ', ' . $_ENV['KNPU_LOREM_IPSUM_BAR'];
    $event->setData($data);
}
</pre>

**Comment by Nyholm** [comment]
This recipe just add default values (and not even the one promoted by the bundle). 

Flex is smart enough to install this bundle without a recipe.

**Pull request closed without merging**